data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
+			`	You approach one of the nearby Wanderers and ask if they could direct you to Tema'a Iriket, the terraforming director. They don't appear to understand the Hai language, but they seem to recognize the name and gesture for you to wait in the spaceport, before heading off in a different direction, presumably to fetch Iriket for you.`
+
+
+
+mission "Terraforming 14"
+	name "Field Trip"
+	description "Take Tema'a Iriket and Amy to <destination> to show her a world where the Wanderers have finished their work."
+	source "Desi Seledrak"
+	destination "Var' Kayi"
+	passengers 2
+	blocked "You need <capacity> in order to take on this mission."
+	to offer
+		has "Terraforming 13: done"
+	on offer
+		conversation
+			`It is not long before Iriket arrives. You introduce Amy to her, and explain why you have brought her here. "I would be [glad, honored] to answer any questions about our work," she agrees.`
+			`	What follows is an extensive discussion about how the Wanderers go about planning the terraforming process for a world. From the earliest considerations as to what the goal for each planet should be to the role of their artificial minds in helping to monitor and guide the process in the long term. Your translator has some difficulty with the more technical aspects of the discussion, which leads Iriket to rephrase some things many times. Throughout it all, Amy is taking copious notes. Iriket also explains some of what they have been doing to improve the climate on this world.`
+			`	"The gases [released, produced] by the factories will help the atmosphere retain more heat, which will in turn begin to melt the glaciers and return water to the [ecosystem, environment]. Over time, this will [mix, interact] with the abundant ash, helping to [restore, rejuvenate] the soils all over this world. In time, it will become [fruitful, bountiful] again."`
+			`	"Won't that take hundreds or thousands of years?" Amy asks.`
+			`	"Indeed," says Iriket. "But, to attempt to [accelerate, advance] the process by more [direct?] means would [harm, deteriorate] the product. While some worlds may [benefit from, require] a more [extreme, radical] change to begin their restoration, we do not believe it is [necessary, appropriate] here. It is [important, our goal] that a world at the conclusion of our work not only be [pristine, healthy] in that moment, but that it be [sustainable?] without additional [intervention, support]."`
+			`	Eventually, Amy asks "Are there any worlds where you have completed your work that I could see?" Iriket agrees and asks you to take them to <destination>.`
+			choice
+				`	"Sure."`
+					accept
+				`	"Sorry, I need to go elsewhere."`
```

Comment:
And you just leave her in the middle of alien space, taking the only translating device? That sounds like fun.

Reply:
I removed the choice.